app/controllers/audits/allocations_controller.rb

```diff
@@ -37,13 +38,20 @@ def destroy
 
   private
 
+    PAGE_SIZE = 100
```

Comment:
I've partially addressed this with 88158a7fbeb54d7fa188b2b67e17a954cfd83568 by moving the default value to the filter object so that it is accessible in this context.

I'm generally uncomfortable with the concern of paging being mixed in with filtering but I didn't want to address that here and instead think about it with https://trello.com/c/seNXaaRp